content/documentation/zooming-images-maps.md

```diff
@@ -4,4 +4,77 @@ weight: 4900
 type: essay
 ---
 
+## Zooming Images
+
 TK
+
+## Maps
+
+<div class="box warning">
+
+- Basic mapping functionality was added into Quire early on, but hasn't been used in production on any projects yet and so remains very rough. Maps are unlikely to be a supported feature moving forward as they can be very demanding and beyond what we’re able to support for the Quire community at large. Our hope though is that we’ll be able to demonstrate some working examples in the future to at least act as a guide if you wish to pursue your own maps. In the meantime, you’re welcome to make use of the map functionality as it exists, but proceed through this section at your own risk.
```

Comment:
Adjusted some of the phrasing:

Basic mapping functionality was added into Quire early on but hasn't been used in production on any projects yet, and so remains very rough. Maps are unlikely to be a supported feature moving forward as they can be very demanding and beyond what we can support for the Quire community at large. In the future, we hope to demonstrate some working examples that can act as a guide if you wish to pursue maps yourself. In the meantime, you're welcome to make use of the map functionality as it exists but proceed through this section with caution.